axelor-account/src/main/java/com/axelor/apps/account/web/AnalyticDistributionLineController.java

```diff
@@ -43,4 +48,31 @@ public void validateLines(ActionRequest request, ActionResponse response) {
           "The distribution is wrong, some axes percentage values are higher than 100%");
     }
   }
+
+  public void manageNewAnalyticDistributionLine(ActionRequest request, ActionResponse response)
+      throws AxelorException {
+    MoveLine moveLine = request.getContext().getParent().asType(MoveLine.class);
+    if (moveLine != null)
+      response.setValue(
+          "analyticJournal",
+          Beans.get(AccountConfigService.class)
+              .getAccountConfig(moveLine.getAccount().getCompany())
+              .getAnalyticJournal());
+    if (moveLine.getDate() != null) {
+      response.setValue("date", moveLine.getDate());
+    } else {
+      response.setValue(
+          "date", Beans.get(AppBaseService.class).getTodayDate(moveLine.getAccount().getCompany()));
+    }
```

Comment:
all code in controller methods must be called inside a try {} catch {}, so the method must never throw exceptions